Gemfile

```diff
@@ -12,7 +12,7 @@ group :development, :test do
   gem 'guard'
   gem 'guard-rspec'
   gem 'growl'
-  gem 'rb-fsevent'
+  gem 'rb-fsevent', '~> 0.9'
```

Comment:
The pervious version of `rb-fsevent` is not working on Yosemite, so I updated it to 0.9.